tests/test_gpu_openmp.py

```diff
@@ -194,7 +194,7 @@ def test_timeparallel_reduction(self):
         assert not tree.root.pragmas
         assert len(tree[1].pragmas) == 1
         assert tree[1].pragmas[0].value ==\
-            'omp target teams distribute parallel for collapse(3) reduction(+:f[0])'
+            'omp target teams distribute parallel for collapse(3) reduction(+:f[0:1])'
```

Comment:
This should have been caught by ` if i.is_Indexed:` and still be `f[0]` since it's a single index. Is it from my PR?

Reply:
no, from mine, I was just handling both cases (number and symbol) homegenously. But I've now rolled back to previous behaviour